apps/webpack.js

```diff
@@ -122,6 +122,7 @@ var storybookConfig = _.extend({}, baseConfig, {
     new webpack.ProvidePlugin({React: 'react'}),
     new webpack.DefinePlugin({
       IN_UNIT_TEST: JSON.stringify(false),
+      IN_STORYBOOK: JSON.stringify(true),
```

Comment:
I'm surprised storybook doesn't already provide this somehow, but I looked around and it doesn't seem to. :+1: